pkgs/development/libraries/glib/default.nix

```diff
@@ -151,6 +151,8 @@ stdenv.mkDerivation (finalAttrs: {
     "-Dgtk_doc=${lib.boolToString buildDocs}"
     "-Dnls=enabled"
     "-Ddevbindir=${placeholder "dev"}/bin"
+  ] ++ lib.optionals (!stdenv.isLinux) [
+    "-Dlibelf=disabled"
```

Comment:
This controls a few ifdefs in [gio/gresource-tool.c](https://gitlab.gnome.org/GNOME/glib/-/blob/main/gio/gresource-tool.c) only.

Reply:
The feature allows dumping gresources embedded into ELF files (e.g. `gresource list result/lib/libgtk-3.so.0`). Not sure how important that is on Darwin when they use Mach-O instead, probably only if someone is debugging Linux programs on Mac.